stack/stack.go

```diff
@@ -34,33 +36,69 @@ func getTemplateFromFile(context *context.Context) (string, string) {
 // This function uses CreateStackInput variable to create Stack.
 func createStack(templateStruct cloudformation.CreateStackInput, session *session.Session) {
 	api := cloudformation.New(session)
-	api.CreateStack(&templateStruct)
+	_, err := api.CreateStack(&templateStruct)
+	if err != nil {
+		println("Error creating stack", err.Error())
+	}
 }
 
 // This function uses all functions above and session to create Stack.
 func NewStack(context *context.Context) {
 	template, stackName := getTemplateFromFile(context)
 	templateStruct := createStackInput(context, &template, &stackName)
+
 	tokenError := mysession.UpdateSessionToken(context.Config.DefaultProfile, context.Config.DefaultRegion, context.Config.DefaultDurationForMFA, context)
 	if tokenError != nil {
 		context.Logger.Error(tokenError.Error())
 	}
-	session, createSessionError := mysession.CreateSession(context, context.Config.DefaultProfile, &context.Config.DefaultRegion)
+	currentSession, createSessionError := mysession.CreateSession(context, context.Config.DefaultProfile, &context.Config.DefaultRegion)
 	if createSessionError != nil {
 		context.Logger.Error(createSessionError.Error())
 	}
-	createStack(templateStruct, session)
+
+	if *context.CliArguments.Progress {
+		conn, err := notificationservice.GetRemoteSink(context, currentSession)
+		if err != nil {
+			context.Logger.Error("Error getting remote sink configuration: " + err.Error())
```

Comment:
Add friendly information that user can use the following command to set it up.

Reply:
It is already in the error body since it is a custom error

Reply:
Can you provide an example how does it look like?

Reply:
```ERROR: Error getting remote sink configuration: remote sink has not been configured, run 'perun setup-remote-sink' first. If You have done it already, wait for aws sink configuration```

Reply:
Then it is awesome, let's leave it as is.